src/lib/serviceRoutinesV2/putEntityAttributeValue.cpp

```diff
@@ -79,6 +81,15 @@ std::string putEntityAttributeValue
     }
   }
 
+  if (ciP->httpStatusCode == SccConflict)
+  {
+    ErrorCode   ec("TooManyResults", "There is more than one entity that match the update. Please refine your query.");
```

Comment:
This is what we have right now:

    getEntityAttributeValue.cpp: "There is more than one entity with that id - please refine your query"
    Entity.cpp:                  "There is more than one entity with that id. Refine your query."
    Attribute.cpp:               "There is more than one entity with that id. Refine your query."

I still prefer "More than one matching entity".

A decision to be made ...